### PR TITLE
Dominian Naval Ship

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
WIP. Adds a Dominian naval third party ship.

Note: There's no reason for this to be failing the waypoint tests to my knowledge. I included four waypoints.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Comment by TheDanseMacabre** [comment]
!wip

File: maps/away/ships/dominia/dominian_corvette.dm [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	icon_state = "shuttle"
```
Comment (WickedCybs) [inline maps/away/ships/dominia/dominian_corvette.dm:29]: Make sure you use ship, not shuttle. There's colours you can choose from in the dmi if any seem appropriate.

File: maps/away/ships/dominia/dominian_corvette.dm [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	moving_state = "shuttle_moving"
```
Comment (WickedCybs) [inline maps/away/ships/dominia/dominian_corvette.dm:30]: Ditto.

**Comment by TheDanseMacabre** [comment]
!review

**Comment by WickedCybs** [comment]
Check if you have any repeated navpoints. That might be the problem behind the error.

**Comment by TheDanseMacabre** [comment]
> Check if you have any repeated navpoints. That might be the problem behind the error.

I don't think I do.

**Review by WickedCybs — APPROVED** [review]
It works fine so I approved it. Think we should make sure the error doesn't show up on any other PRs if this is merged though.

**Review by WickedCybs — CHANGES_REQUESTED** [review]
actuall you need Weapon Entry points (not navpoints) a,sko j i figured this ouit not wildkins lol your welcome

**Comment by TheDanseMacabre** [comment]
Thabnjks Cybs and NOTTT wildkins.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by WickedCybs — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**